DATA/SConscript

```diff
@@ -153,6 +153,15 @@ crosstalk = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_crosstalk'),
                          getCertifyCmd('crosstalk')])
 env.Alias('crosstalk', crosstalk)
 
+#PTC
+ptc = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_ptc'),
+                         os.path.join(REPO_ROOT, "calib", 'v00', "ptc")], crosstalk,
+                        [getPipeTaskCmd('ptc', exposureDict['ptcExposurePairs'],
+                                        'measurePhotonTransferCurve.yaml'),
```

Comment:
As this inherits the default cp_pipe pipeline file, this is going to run with the POLYNOMIAL fit, not the FULLCOVARIANCE.  Since the downstream code relies on the FULLCOVARIANCE, should that be the default?

Reply:
I changed `ptcFitType` to `FULLCOVARIANCE` in `measurePhotonTransferCurve.yaml` in `cp_pipe` and verified that `ci_cpp_gen3` successfully builds.